.travis.yml

```diff
@@ -14,6 +14,7 @@ install:
   - python bootstrap.py
   - bin/buildout annotate
   - bin/buildout -Nq
+  - cat bin/test
```

Comment:
Thanks for reviewing!
Actually annotate doesn't give you all the versions. 
In this instance I think I wanted to know what version collective.js.cycle2 we were using.
It is given by `cat bin/test` and not by `bin/buildout annotate`

Search for `collective.js.cycle2` in the [raw travis log](https://s3.amazonaws.com/archive.travis-ci.org/jobs/89660221/log.txt) to see what I mean.

There might be other ways than cat bin/test, but it's the best I've got so far!